x/stakeibc/types/params.go

```diff
@@ -27,7 +27,7 @@ var (
 	DefaultIBCTransferTimeoutNanos     uint64 = 1800000000000 // 30 minutes
 	DefaultSafetyNumValidators         uint64 = 35
 	DefaultSafetyMaxSlashPercent       uint64 = 10
-	DefaultValidatorSlashQueryInterval uint64 = 100_000_000_000 // denominated in number of native tokens
+	DefaultValidatorSlashQueryInterval uint64 = 1 // denominated in percentage of TVL (1 => 5%)
```

Comment:
(1 => 5%)???
Is this a typo and it was supposed to be (5 => 5%)

Reply:
yup sorry typo. fixing now